modules/ossm-supported-configurations.adoc

```diff
@@ -45,8 +45,7 @@ For additional information about {SMProductName} lifecycle and supported configu
 
 * Jaeger agent as a sidecar is the only supported configuration for Jaeger. Jaeger as a daemonset is not supported for multitenant installations or OpenShift Dedicated.
 
-[id="ossm-supported-configurations-adapters_{context}"]
-== Supported Mixer adapters
+[id="ossm-supported-configurations-webassembly_{context}"]
+== Supported WebAssembly module
 
-* This release only supports the following Mixer adapter:
-** 3scale Istio Adapter
+* 3scale WebAssembly is the only supported WebAssembly module.
```

Comment:
"3scale WebAssembly is the only **supported** WebAssembly module."  I could see people taking this literally, let's change this to "3scale WebAssembly is the only **provided** WebAssembly module.  You can create custom WebAssembly modules."

Reply:
Done